src/open_inwoner/scss/components/Grid/Grid.scss

```diff
@@ -37,16 +37,14 @@
       }
 
       &.banner__text {
-        .container:not(:has(.notifications .notification)) ~ .container & {
-          $image-height-mobile: 140px;
-          margin-top: -$image-height-mobile;
-        }
-
-        margin-top: 0;
+        $image-height-mobile: 140px;
+        margin-top: -$image-height-mobile;
         padding: var(--spacing-large);
         padding-bottom: 0;
         box-sizing: border-box;
         width: 100%;
+        border-top-left-radius: var(--border-radius);
+        border-top-right-radius: var(--border-radius);
         background-color: rgba(255, 255, 255, 0.6); /* fallback */
         @supports (background-color: color-mix(in srgb, white, transparent)) {
```

Comment:
@pi-sigma I have to change this color to white instead of a transparency (for accessibility reasons).